api/pkg/cmd/stack.go

```diff
@@ -1,65 +1,61 @@
 package cmd
 
 import (
+	"context"
+	"encoding/json"
+
 	"github.com/chanzuckerberg/happy/api/pkg/dbutil"
 	"github.com/chanzuckerberg/happy/shared/model"
 	"github.com/pkg/errors"
-	"gorm.io/gorm/clause"
+	"github.com/sirupsen/logrus"
 )
 
-type Stack interface {
-	CreateOrUpdateAppStack(model.AppStackPayload) (*model.AppStack, error)
-	GetAppStacks(model.AppStackPayload) ([]*model.AppStack, error)
-	DeleteAppStack(model.AppStackPayload) (*model.AppStack, error)
+type StackIface interface {
```

Comment:
https://go.dev/doc/effective_go#interface-names

Reply:
StackGetter, StackSetter, StackDeleter, then combine those to be like StackManager or something

Reply:
I'm hesitant to create StackGetter, StackSetter, and StackDeleter when they're not strictly needed at this point. But I'm renaming StackIface to StackManager